libraries/chain/include/eosio/chain/finality/finalizer.hpp

```diff
@@ -74,8 +74,8 @@ namespace eosio::chain {
       ///
       /// Version 0: Spring 1.0.0 RC2 - File has fixed packed sizes with inactive safety info written to the end
       ///                               of the file. Consists of [finalizer public_key, FSI]..
-      /// Version 1: Spring 1.0.0 RC3 - Variable length FSI with inactive safety info written at the beginning of
-      ///                               the file. Uses crc32 checksum to verify data on read.
+      /// Version 1: Spring 1.0.0 RC3 - File has inactive FSIs written at the beginning of the file. Uses crc32
```

Comment:
The version 1 comment should also mention it removes `votes_forked_since_latest_strong_vote` from the version 0 FSI and replaces it with `other_branch_latest_time`.